frontend/components/AddFundsThroughBridge/useGenerateAddFunds.tsx

```diff
@@ -0,0 +1,126 @@
+import { uniq } from 'lodash';
```

Comment:
not specific to this PR, in general should we move towards using specific module instead of the large lodash module? 
(eg: https://www.npmjs.com/package/lodash.kebabcase)